src/App.tsx

```diff
@@ -1,61 +1,142 @@
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+import fetchedUsers from './api/users';
+import fetchedTodos from './api/todos';
+import { User } from './Types';
+import { TodoList } from './components/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const todosWithAssociatedUsers = fetchedTodos.map(todo => ({
```

Comment:
Nice naming here. Some may say that it's too long, but I can call it readable and easy to grasp :)